src/Component/Editors/Editor.js

```diff
@@ -0,0 +1,83 @@
+import { validateNew, validateString, vaildateArray } from '../../utils/validation.js'
+
+export default function Editor({ $target, initialState = { title: '', content: '' }, onEditing, onClick }) {
+    validateNew(new.target)
+
+    const $editor = document.createElement('div')
+    $editor.className = 'editor'
+
+    const $subdocumentList = document.createElement('div')
+
+    $target.appendChild($editor)
+    $target.appendChild($subdocumentList)
+
+    this.state = initialState
+    validateString(this.state)
```

Comment:
```suggestion
    validateString(initialState)
    this.state = initialState
```
저는 이런 방식으로 검사해줬는데, 어떤 방식이 더 적절한지 궁금하네요!

Reply:
state로 변경해주기 전에 검사하는 지윤님 방식이 더 좋은 것 같네요.. 수정해야겠습니다!!